test-app/tests/test-helper.js

```diff
@@ -4,6 +4,9 @@ import * as QUnit from 'qunit';
 import { setApplication } from '@ember/test-helpers';
 import { setup } from 'qunit-dom';
 import { start } from 'ember-qunit';
+import { enableTimeout } from 'ember-cli-flash/test-support';
+
+enableTimeout();
```

Comment:
The test app needs to dogfood the `enableTimeout` helper since it tests timeout functionality of the addon itself.